packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -79,7 +84,7 @@ export class VerticalStackedBarChartBase extends React.Component<
   IVerticalStackedBarChartState
 > {
   private _points: IVerticalStackedChartProps[];
-  private _dataset: IDataPoint[];
+  private _dataset: IVerticalStackedBarDataPoint[];
```

Comment:
>private [](http://example.com/codeflow?start=2&length=7)

can you try with an example using IDataPoint in the input data.
Does it throw a warning or lint error or something

Reply:
This is not exposed in examples.

Reply:
Then this interface should not be exposed publicly in the api.md file. It only confuses the user. Can you log a task for it.

Reply:
Created the task for it.